### DISCUSSION TITLE
UI Enhancement

### DISCUSSION THREAD

**Discussion by shashank-iter** [Rick-mad-lab/Todolist#61] [General]
I saw the website and I felt that it needs a UI upgrade with some custom designed background and Glassmorphism.  I can make the UI more user-friendly and tempting just by redesigning a few components. @Sharanya1-lab, I have mentioned my idea about the UI re-design, can I go for it?

**Comment by sharanya-mitra** [comment]
We are working on it do you want to change anything in this New UI 
- [new UI](https://github.com/Rick-mad-lab/Todolist/blob/UX/UI-Desktop/User%20Experience/Design.md)
> we are working on two different UI One for [Desktop](https://github.com/Rick-mad-lab/Todolist/tree/UX/UI-Desktop) and One for [Mobile](https://github.com/Rick-mad-lab/Todolist/tree/UX/UI-Mobile)

**Comment by sharanya-mitra** [comment]
@Rahul6918 can you explain your Ideas to @shashank-iter.

**Comment by shashank-iter** [comment]
![Screenshot 2022-10-02 010905](https://user-images.githubusercontent.com/102979508/193464820-57b72f73-c4c0-4a36-a6c2-0f2c0e107445.png)

I recently designed a UI of a note taking app. I can design something like this.

**Comment by sharanya-mitra** [comment]
yep but make it more classic not too fancy!
😃

**Comment by shashank-iter** [comment]
Can we go for some video background, in which a video of a classic library
will play ?

**Comment by sharanya-mitra** [comment]
hmm, make simple UI and share the SS.
